sample_apps/run_example.sh

```diff
@@ -409,6 +679,88 @@ function print_signed_claim() {
    popd > /dev/null 2>&1
 }
 
+# ###########################################################################
+# Step 7 in instructions.md: Manual policy generation
+# You run either this step, or Step 8, below.
+# Author the policy for the security domain and produce the signed claims
+# that the apps need. This is just a wrapper function to drive the execution
+# of sub-steps, each of which is implemented by a minion function.
+# ###########################################################################
+function author_policy_for_OE() {
+    pushd "${PROV_DIR}" > /dev/null 2>&1
+
+    run_cmd
+
+    construct_policyKey_platform_is_trusted
+
+    produce_signed_claims_for_vse_policy_statement
+
+    combine_policy_stmts
+
+    print_policy
+
+    popd > /dev/null 2>&1
+}
+
+# ###########################################################################
+# Step 8 in instructions.md: Use Policy Generator
+# You run either this or step (7).
+# ###########################################################################
+function generate_policy_for_OE() {
+    pushd "${PROV_DIR}" > /dev/null 2>&1
+
+    edit_policy_file_OE
+    run_policy_generator_OE
+
+    popd > /dev/null 2>&1
+}
+
+# ###########################################################################
+# Edit the policy.json file to replace trusted measurements of the
+# "measurements" property with expected measurements from
+# 'make dump_mrenclave'
+# ###########################################################################
+function edit_policy_file_OE() {
+    pushd "${EXAMPLE_DIR}" > /dev/null 2>&1
+
+    policy_json_file="./oe_policy.json"
+    local policy_json_file_old="${policy_json_file}.old"
+    run_cmd cp -p ${policy_json_file} ${policy_json_file_old}
+
+    # Grab the 'mrenclave' term from 'make' output
+    local mrenclave=""
+    set -x
+    mrenclave=$(make dump_mrenclave | grep "^mrenclave" | cut -f2 -d'=')
+    set +x
+
+    # Update json file using 'jq' tool, to replace value of the
+    # 'measurement' key with generic term 'mrenclave'. Then do a text
+    # replacement of this with the mrenclave grabbed above.
+    set -x
+    Jq=$(command -v jq)
+    ${Jq} '.measurements = [ "mrenclave" ]' "${policy_json_file}"    \
```

Comment:
This introduces a dependency on external tool `jq` to do this in-place edit of the json file.

I found no other more reliable way.

I have doc'ed this dependency in the md-file [here, in the README.md](https://github.com/vmware-research/certifier-framework-for-confidential-computing/tree/agurajada/67-run_example_for-oe-app/sample_apps/simple_app_under_oe).

---
Setup the simple app by editing the policy JSON file and use policy generator: For this interface, you will need to install the jq utility for editing the JSON file. $ sudo apt-get install -y jq